collect/collect.go

```diff
@@ -348,18 +447,18 @@ func (i *InMemCollector) processSpan(sp *types.Span) {
 		}
 
 		trace = &types.Trace{
-			APIHost:    sp.APIHost,
-			APIKey:     sp.APIKey,
-			Dataset:    sp.Dataset,
-			TraceID:    sp.TraceID,
-			StartTime:  time.Now(),
-			SendBy:     time.Now().Add(timeout),
-			SampleRate: sp.SampleRate, // if it had a sample rate, we want to keep it
+			APIHost:     sp.APIHost,
+			APIKey:      sp.APIKey,
+			Dataset:     sp.Dataset,
+			TraceID:     sp.TraceID,
+			ArrivalTime: time.Now(),
+			SendBy:      time.Now().Add(timeout),
```

Comment:
For clarity in the resulting values, consider capturing `time.Now` once above line 449:
```go
now := time.Now()
```
Then, use `now` whenever we're trying to refer to the current time.

Reply:
Good idea.